transmonee_dashboard/src/transmonee_dashboard/pages/base_page.py

```diff
@@ -818,17 +788,15 @@ def get_target_query(data, indicator, dimension="Sex", target_code="Total"):
         query_item = []
         for item in disag:
             item_total = []
```

Comment:
not a change in the logic but a rearrange of the code. I would use a different variable name @lbassil, just to make more clear that the return of `get_disag_total` is not a dataframe `df` but the total code for a dimension (previously devised as a potential a list of codes meaning total)